CONTRIBUTING.md

```diff
@@ -265,8 +265,9 @@ See [`apm-agent-plugins/README.md`](apm-agent-plugins/README.md)
 
 HTML Documentation is generated from text files stored in `docs` folder using [AsciiDoc](http://asciidoc.org/) format.
 The `configuration.asciidoc` file is generated from running `co.elastic.apm.agent.configuration.ConfigurationExporter`
-(e.g. via `./mvnw -Dsurefire.failIfNoTests=false -Dtest=ConfigurationExporterTest -pl apm-agent -am clean test`). All the other asciidoc text files
-are written manually.
+(e.g. via `./mvnw -Dsurefire.failIfNoTests=false -Dtest=ConfigurationExporterTest -pl apm-agent -am clean test`
+or on Windows `.\mvnw '-Dsurefire.failIfNoTests=false' '-Dsurefire.failIfNoSpecifiedTests=false' -Dtest=ConfigurationExporterTest -pl apm-agent-builds/apm-agent -am clean test`). 
```

Comment:
@jackshirazi what version of windows do you use?
System properties with dots not recognized properly.

Can you try to run with double quotes? It is working properly on my windows as with single quotes.
`.\mvnw "-Dsurefire.failIfNoTests=false" "-Dsurefire.failIfNoSpecifiedTests=false" -Dtest=ConfigurationExporterTest -pl apm-agent-builds/apm-agent -am clean test`

Reply:
double quotes works. I'm on Windows 11, but I think it's more about the command window used than the windows version. Let's go with the double quotes since that works everywhere

Reply:
```suggestion
or on Windows `.\mvnw "-Dsurefire.failIfNoTests=false" "-Dsurefire.failIfNoSpecifiedTests=false" -Dtest=ConfigurationExporterTest -pl apm-agent-builds/apm-agent -am clean test`). 
```

Reply:
fixed